xlators/cluster/ec/src/ec-inode-write.c

```diff
@@ -710,13 +710,29 @@ ec_setxattr(call_frame_t *frame, xlator_t *this, uintptr_t target,
     ec_cbk_t callback = {.setxattr = func};
     ec_fop_data_t *fop = NULL;
     int32_t error = ENOMEM;
+    data_t *dict_data = NULL;
+    char *mask_key = "user.readmask";
```

Comment:
create a macro like we have with "trusted.ec.heal". Name it "trusted.ec.readmask"